packages/css-data/src/property-parsers/__snapshots__/shadows.test.ts.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@TrySound what's your opinion on shapshots, I wonder if we had switched to inline snapshots

Reply:
yep, we use only inline snapshots